sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.1 (2025-06-22)
```

Comment:
why it's not 1.3.0?

Reply:
it should be released today or on 22th?

Reply:
I am not sure what is the best date to set here . I cannot guarantee I can make it today. The only think I know for sure is the deadline. Is it ok to set there e.g. today but if I cannot make it release it tomorrow? 
I 'll change to 1.3.0 :)